src/engraving/libmscore/slur.cpp

```diff
@@ -1462,23 +1467,31 @@ SpannerSegment* Slur::layoutSystem(System* system)
 
             _up = !(startCR()->up());
 
+            // Check if multiple voices
+            bool multipleVoices = false;
             Measure* m1 = startCR()->measure();
-            if (c1 && c2 && !c1->isGrace() && isDirectionMixture(c1, c2)) {
+            ChordRest* cr2 = endCR() ? endCR() : startCR();
+            while (m1 && m1->tick() <= cr2->tick()) {
+                if ((m1->hasVoices(startCR()->staffIdx(), tick(), ticks() + (cr2 ? cr2->ticks() : startCR()->ticks())))
```

Comment:
Sorry, another tiny thing. Since `endCR()` is never nullptr here (see line 1441), the line `ChordRest* cr2 = endCR() ? endCR() : startCR();` is not necessary. Just use `endCR()` instead of `cr2`. 

For the same reason, `(cr2 ? cr2->ticks() : startCR()->ticks())` can be replaced with simply `endCR()->ticks()`.